### PR TITLE
Add coded race and ethnicity

### PR DESCRIPTION
**Pull request by saulakravitz**
- added coded race and ethnicity composition
- VRDR references VRCL input and coded race and ethnicity profiles
- Deleted VRDR coded race and ethnicity profiles.  Renamed VRCL versions  (no longer 'new').
- pre-coordinated input/coded race and ethnicity with mother and father linkage

This is a first cut...Probably needs more work.

### CODE REVIEW

**Comment by rpassas** [comment]
Content transitions still refer to `InputRaceAndEthnicityNew`, which will lead to broken links

**Comment by rpassas** [comment]
Not sure if this is intentional or an oversight - `RequireMetaProfile` is just a placeholder at the moment and still includes `-New` profiles

File: input/fsh/VRCL/harmonized-profiles/ObservationCodedRaceAndEthnicity.fsh [hunk 1]
```diff
@@ -8,7 +8,7 @@ This profile is only for use in a Vital Records Exchange. Systems looking to exc
 
 Note:  NCHS stopped providing RaceRecode40 data in 2022."
 * insert RequireMetaProfile(Profile: CodedRaceAndEthnicityNew)
-* code = ObservationsCS#codedraceandethnicity
+* code from ValueSetCodedRaceAndEthnicityPersonVitalRecords (required) //  code = ObservationsCS#codedraceandethnicity
 * value[x] 0..0
```
Comment (rpassas) [inline input/fsh/VRCL/harmonized-profiles/ObservationCodedRaceAndEthnicity.fsh:12]: There were some minor differences between this version and the published version. Here there is:
- a constraint on focus 0..* --> 0..1 and a constraint on referenced profiles
- no effective[x]

**Comment by saulakravitz** [comment]
> Content transitions still refer to `InputRaceAndEthnicityNew`, `CompositionCodedRaceAndEthnicityNew` (any -New that was removed), which will lead to broken links

We'll clean up all of the new stuff in one pass at some point...

File: input/fsh/VRCL/harmonized-profiles/ObservationCodedRaceAndEthnicity.fsh (hunk 1 above)
Comment (saulakravitz) [reply input/fsh/VRCL/harmonized-profiles/ObservationCodedRaceAndEthnicity.fsh:12]: What should it be?

**Pull request merged**